src/test/java/unstable/ContextMenuUnstableTests.java

```diff
@@ -0,0 +1,100 @@
+package unstable;
+
+import guitests.UITest;
+import javafx.application.Platform;
+import javafx.scene.control.ContextMenu;
+import javafx.scene.control.MenuItem;
+import javafx.scene.input.KeyCode;
+import org.junit.Before;
+import org.junit.Test;
+import ui.components.FilterTextField;
+import ui.components.IssueListView;
+import ui.listpanel.ListPanel;
+import ui.listpanel.ListPanelCell;
+import java.lang.reflect.*;
+
+import static org.junit.Assert.assertEquals;
+
+public class ContextMenuUnstableTests extends UITest {
+
+    private static final String PANEL_IDENTIFIER = "#dummy/dummy_col0";
+
+    @Before
+    public void setup() {
+        Platform.runLater(stage::show);
+        Platform.runLater(stage::requestFocus);
+
+        FilterTextField filterTextField = find(PANEL_IDENTIFIER + "_filterTextField");
+        filterTextField.setText("");
+        Platform.runLater(filterTextField::requestFocus);
+
+        click(PANEL_IDENTIFIER + "_filterTextField");
+        push(KeyCode.ENTER);
+    }
+
+    /**
+     * Tests selecting "Mark all below as read" and "Mark all below as unread" context menu items
+     */
+    @Test
+    public void markAllBelowAsReadUnread_twelveIssuesInListView_issuesCorrectlyMarkedReadUnread() {
+        ListPanel issuePanel = find(PANEL_IDENTIFIER);
+        try {
+            Field listViewField = ListPanel.class.getDeclaredField("listView");
+            listViewField.setAccessible(true);
+            IssueListView listViewValue = (IssueListView) listViewField.get(issuePanel);
+
+            // scrolls to the end of the panel
+            listViewValue.scrollAndShow(12);
+        } catch (NoSuchFieldException e) {
+            throw new RuntimeException(e);
+        } catch (IllegalAccessException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
These two catch blocks can be merged. In any case, catching and rethrowing exceptions inside tests is meaningless. Just remove the try clause.